src/test/resources/net/sf/jabref/importer/fileformat/RepecNepImporterTest1.bib

```diff
@@ -0,0 +1,12 @@
+@TechReport{,
+  Title                    = {Innovation races: An experimental study on strategic research activities},
+  Year                     = {2005},
+  Month                    = {10},
+
+  Abstract                 = {In an experimental setting, firms in a duopoly market engage in a patent tournament and compete for profit-enhancing product advancements. The firms generate income by matching exogenously defined demand preferences with an appropriately composed product portfolio of their own. Demand preferences are initially unknown and first need to be revealed by an investigation of the possible product variations. The better firms approximate demand preferences, the higher their profits. In the ensuing innovation race, firms interact through information spillovers resulting from the imperfect appropriability of research successes. In the random period of the experiment, the continuity of the search process is disturbed by an exogenous shock that affects both the supply and demand side and again spurs research competition. Firms may henceforth explore an enlarged product space in attempting to match the equally modified demand preferences. In our analysis, we explore the behavioural regularities of agents who are engaged in innovation activities. As a key element we test to what extend relative economic performance exercises a stimulating effect on the implementation of innovation and imitation strategies.},
+  author = {Uwe Cantner and Andreas Nicklisch and Torsten Weiland},
+  institution = {University of Jena, Faculty of Economics and Max-Planck-Institute for Research into Collective Goods and Max-Planck-Institute for Economics, Strategic Interactions Group},
+  Jel                      = {D81 O31},
```

Comment:
https://ideas.repec.org/j/
Here are top classification for JEL. The D for example means, that it is classified as Microeconomics. But I'm not sure what the meaning of the number behind is meaning.